system-validation-test/Gemfile.lock

```diff
@@ -39,9 +39,9 @@ GEM
     multi_json (1.15.0)
     multi_test (1.1.0)
     multi_xml (0.6.0)
-    nokogiri (1.13.9-x86_64-darwin)
+    nokogiri (1.13.10-x86_64-darwin)
```

Comment:
Is this supposed to be `1.13.10` or `1.13.10-x86_64-darwin)` I only ask because https://github.com/nasa/Common-Metadata-Repository/pull/1813 is changing it to the former